application/controllers/UserManagementController.php

```diff
@@ -775,12 +775,21 @@ public function actionImportUsers(string $importFormat = 'csv'): string
         }
         $created = [];
         $updated = [];
-
         foreach ($aNewUsers as $aNewUser) {
             $oUser = User::model()->findByAttributes(['users_name' => $aNewUser['users_name']]);
 
             if ($oUser  !== null) {
                 if ($overwriteUsers) {
+                    /* Check permission to edit this user */
+                    if (!$oUser->canEdit() || $aUser['uid'] == App()->user->id) {
+                        Yii::app()->setFlashMessage(sprintf(gT("You don't have permission to edit user %s."), $aNewUser['users_name']), 'warning');
+                        continue;
+                    }
+                    /* Check permission to edit self */
+                    if ($aUser['uid'] == App()->user->id) {
```

Comment:
Will this lines in this if statement ever be executed?
If `$aUser['uid'] == App()->user->id` the script will enter in the line above, right?

Reply:
Oh yes : i just want to send a different message burt don't remove the or part …